<?xml version="1.0" ?>
<launch>
    <arg name="camera_name" default="realsense" />
    <arg name="camera_info" default="realsense_backup2" />


       <node name="acrv_realsense_ros" pkg="acrv_realsense_ros" type="acrv_realsense_ros_node" output="screen" respawn="false">
           <param name="serial_number" type="string" value="617206000061" /> -->
           <param name="rotate_image_180" type="bool" value="true" />
           <param name="rotate_image_180" type="bool" value="false" />
           <param name="camera_name" type="string" value="$(arg camera_name)" />
           <param name="rgb_hd_camera_info_path" type="string" value="/home/apc-cameras/ros_ws/src/acrv_apc_2017/acrv_cameras/acrv_realsense_ros/cfg/$(arg camera_info)/rgb_hd_camera_info.yaml" />
           <param name="depth_camera_info_path" type="string" value="/home/apc-cameras/ros_ws/src/acrv_apc_2017/acrv_cameras/acrv_realsense_ros/cfg/$(arg camera_info)/depth_camera_info.yaml" />
           <param name="rgb_hd_camera_info_path" type="string" value="/home/apc/ros_ws/src/acrv_apc_2017/acrv_cameras/acrv_realsense_ros/cfg/$(arg camera_info)/rgb_hd_camera_info.yaml" />
           <param name="depth_camera_info_path" type="string" value="/home/apc/ros_ws/src/acrv_apc_2017/acrv_cameras/acrv_realsense_ros/cfg/$(arg camera_info)/depth_camera_info.yaml" />
       </node>

       <node pkg="tf" type="static_transform_publisher" name="$(arg camera_name)_optical_frames"

         args="-0.02393014 -0.00044215 0.00339953 0.00013053  0.0042153   0.00358205 /$(arg camera_name)_depth_optical_frame /$(arg camera_name)_rgb_optical_frame 100" />
</launch>
